### PR TITLE
Fix peeradd

### PR DESCRIPTION
**Pull request by peterbroadhurst**
I found that `addSecureContext` was not reliably causing a working TLS context on startup of the firefly CLI with the e2e test.
It looks like simplifying to `setSecureContext ` instead (without supplying a particular hostname) is the right thing to do.

Note in PR chain with #25

### CODE REVIEW

File: src/routers/api.ts [hunk 1]
```diff
@@ -151,7 +149,7 @@ router.post('/messages', async (req, res, next) => {
   }
 });
 
-router.head('/blobs/*', async (req, res, next) => {
+router.head('/blobs/*', async (req: Request, res, next) => {
```
Comment (gabriel-indik) [inline src/routers/api.ts:152]: Are these needed?

File: src/custom.d.ts [hunk 2]
```diff
@@ -16,9 +16,13 @@
 
 export {}
 
-declare global{
+declare global {
   namespace Express {
     export interface Request {
+      params: {
```
Comment (gabriel-indik) [inline src/custom.d.ts:22]: Is this needed?

**Review by gabriel-indik — APPROVED** [review]

**Comment by gabriel-indik** [comment]
It looks like some of the changes introduced in this PR might not have been required. They seem to be workarounds for a specific (probably outdated) typescript compiler installation.

File: src/routers/api.ts (hunk 1 above)
Comment (peterbroadhurst) [reply src/routers/api.ts:152]: I found this was the trick to force the TS compiler to pick up the `config.d.ts` extensions

File: src/custom.d.ts (hunk 2 above)
Comment (peterbroadhurst) [reply src/custom.d.ts:22]: The base express had `string` as the key type, so it wouldn't let me refer to `0`... that was at least what my local setup was complaining about.

**Pull request merged**